src/util/include/util/Color.h

```diff
@@ -63,6 +63,8 @@ struct ColorU8 {
         rhs = val;
         return is;
     };
+
+    constexpr auto isLight() const -> bool { return uint32_t(red) + uint32_t(green) + uint32_t(blue) > 0x180U; }
```

Comment:
I don't know. I just took the constant previously used in the old `BackgroundPainter`s to determine which color set should be used for the rulings.
Changing it would result in some old documents not being displayed the way they used to (the foreground ruling colors could change).